attestation-service/verifier/src/cca/mod.rs

```diff
@@ -81,20 +88,8 @@ impl Verifier for CCA {
             .with_new_session_url(api_endpoint)
             .build()?;
 
-        let mut hasher = Sha384::new();
-        hasher.update(&nonce);
-        hasher.update(&attestation.tee_pubkey.k_mod);
-        hasher.update(&attestation.tee_pubkey.k_exp);
-        let mut hash_of_nonce_pubkey = hasher.finalize().to_vec();
-        hash_of_nonce_pubkey.resize(64, 0);
-
-        log::info!(
-            "HASH(nonce||pubkey):\n\t{}\n",
-            hex::encode(&hash_of_nonce_pubkey)
-        );
-
         let token = evidence.token;
-        let n = Nonce::Value(hash_of_nonce_pubkey.clone());
+        let n = Nonce::Value(report_data.to_vec());
```

Comment:
is it 64 bytes guaranteed? or else should it be resized as 64 bytes?  since the `cca-realm-challenge` is 64 bytes.

Reply:
Is this practical?
- if > 64 bytes: truncate to 64 bytes and raise a `warn`
- if < 64 bytes: padding `\0` to 64 bytes and raise a `warn`

Reply:
> if > 64 bytes: truncate to 64 bytes and raise a warn

should error instead.

> if < 64 bytes: padding \0 to 64 bytes and raise a warn

+1, actually, I am not sure if a warn if needed or not because padding with 0 is usual what we did for it, but I cannot see a problem for a warn here.